maven-plugins/enforcer-maven-plugin/src/main/java/io/helidon/build/maven/enforcer/FileMatcher.java

```diff
@@ -62,6 +62,36 @@ static List<FileMatcher> create(String pattern) {
         return List.of(new ContainsMatcher(pattern));
     }
 
+    /**
+     * Create matcher from git pattern.
+     *
+     * @param pattern pattern to parse
+     * @return matcher that can be matched against a {@link io.helidon.build.maven.enforcer.FileRequest}
+     */
+    static FileMatcher createFromGitPattern(String pattern) {
+        if (pattern.contains("*")) {
```

Comment:
Does the implementation comply with https://git-scm.com/docs/gitignore#_pattern_format ?